modules/react/popup/lib/hooks/usePopupModel.ts

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
-import {createModelHook} from '@workday/canvas-kit-react/common';
+import {createModelHook, ModelExtras, ToEventConfig} from '@workday/canvas-kit-react/common';
 
 import {useDisclosureModel} from '@workday/canvas-kit-react/disclosure';
 import {Placement} from '@workday/canvas-kit-react/popup';
 
 // eslint-disable-next-line no-empty-function
 const noop = () => {};
 
-export const usePopupModel = createModelHook({
+export const _usePopupModel = createModelHook({
```

Comment:
Come back and check the changes in this file.
- [ ] it's ok

Reply:
I did not mean to commit these changes. They are a test of using ASTs in the transformation process to create better types for developers using Canvas Kit. This is an example, but should not be committed.